### PR TITLE
[Workers] websockets revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/learning/using-websockets.md [hunk 1]
```diff
@@ -49,7 +55,7 @@ async function handleRequest(request) {
 }
 ```
 
-Once you've appropriately checked for the `Upgrade` header, you can create a new instance of `WebSocketPair`, which contains _server_ and _client_ WebSockets. One of these WebSockets should be handled by the Workers function (which we'll do shortly), and the other should be returned as part of a `Response` with status 101, indicating the "Switching protocols" response:
+After you have appropriately checked for the `Upgrade` header, you can create a new instance of `WebSocketPair`, which contains server and client WebSockets. One of these WebSockets should be handled by the Workers function, and the other should be returned as part of a `Response` with status `101`, indicating the Switching Protocols response:
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/using-websockets.md]: @lukeed "...indicating the switching protocols response"

Comment (lukeed) [inline products/workers/src/content/learning/using-websockets.md]:
```suggestion
After you have appropriately checked for the `Upgrade` header, you can create a new instance of `WebSocketPair`, which contains server and client WebSockets. One of these WebSockets should be handled by the Workers function and the other should be returned as part of a `Response` with the [`101` status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/101), indicating the request is switching protocols:
```

Comment (lukeed) [reply products/workers/src/content/learning/using-websockets.md]: Added a suggestion. The status text for `101` is "Switching Protocols" which is why this was capitalized originally

File: products/workers/src/content/learning/using-websockets.md [hunk 2]
```diff
@@ -7,35 +7,41 @@ pcx-content-type: concept
 
 <Aside type="warning">
 
-Details such as pricing and limits on Websockets are not yet available. We're looking for developers to experiment with WebSocket support in Cloudflare Workers, but WebSocket support generally remains in early access.
+WebSocket pricing and limits details are currently not available. Developers are encouraged to experiment with WebSocket support in Cloudflare Workers, but WebSocket support generally remains in early access.
 
 </Aside>
 
-WebSockets allow you to communicate in real-time with your Cloudflare Workers serverless functions. In this guide, you'll learn the basics of WebSockets on Cloudflare Workers, both from the perspective of _writing_ WebSocket servers in your Workers functions, as well as connecting to and working with those WebSocket servers as a _client_.
+WebSockets allow you to communicate in real-time with your Cloudflare Workers serverless functions. In this guide, you will learn the basics of WebSockets on Cloudflare Workers, both from the perspective of writing WebSocket servers in your Workers functions, as well as connecting to and working with those WebSocket servers as a client.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/using-websockets.md]:
```suggestion
WebSockets allow you to communicate in real time with your Cloudflare Workers serverless functions. In this guide, you will learn the basics of WebSockets on Cloudflare Workers, both from the perspective of writing WebSocket servers in your Workers functions, as well as connecting to and working with those WebSocket servers as a client.
```

File: products/workers/src/content/learning/using-websockets.md [hunk 3]
```diff
@@ -7,35 +7,41 @@ pcx-content-type: concept
...
-<Aside>
+<Aside type="note">
 
-__Note:__ The `wrangler dev` tool currently does not support connecting to Workers via the WebSocket protocol. Support for WebSockets in `wrangler dev` is tracked in [this GitHub issue](https://github.com/cloudflare/wrangler/issues/1910).
+The `wrangler dev` tool currently does not support connecting to Workers via the WebSocket protocol. Support for WebSockets in `wrangler dev` is tracked in [this GitHub issue](https://github.com/cloudflare/wrangler/issues/1910).
 
 </Aside>
 
-WebSocket servers in Cloudflare Workers allow you to receive messages from a client in real-time. This brief guide will show you how to set up a WebSocket server in Workers.
+WebSocket servers in Cloudflare Workers allow you to receive messages from a client in real-time. This guide will show you how to set up a WebSocket server in Workers.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/using-websockets.md]:
```suggestion
WebSocket servers in Cloudflare Workers allow you to receive messages from a client in real time. This guide will show you how to set up a WebSocket server in Workers.
```

File: products/workers/src/content/learning/using-websockets.md [hunk 4]
```diff
@@ -68,7 +74,7 @@ async function handleRequest(request) {
 }
 ```
 
-An Object is returned from the `WebSocketPair` constructor, with the `0` and `1` keys each holding a `WebSocket` instance as its value. It's common to grab the two WebSockets from this pair using [`Object.values`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) and [ES6 destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), as seen in the below example.
+An Object is returned from the `WebSocketPair` constructor, with the `0` and `1` keys each holding a `WebSocket` instance as its value. It is common to grab the two WebSockets from this pair using [`Object.values`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) and [ES6 destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), as seen in the below example.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/using-websockets.md]:
```suggestion
The `WebSocketPair` constructor returns an Object, with the `0` and `1` keys each holding a `WebSocket` instance as its value. It is common to grab the two WebSockets from this pair using [`Object.values`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) and [ES6 destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment), as seen in the below example.
```

File: products/workers/src/content/learning/using-websockets.md [hunk 5]
```diff
@@ -139,13 +145,13 @@ When the WebSocket interaction is complete, the client can close the connection
...
 
-For an example of this in practice, see the [`websocket-template`](https://github.com/cloudflare/websocket-template) that we provide to get started with WebSockets.
+For an example of this in practice, refer to the [`websocket-template`](https://github.com/cloudflare/websocket-template) to get started with WebSockets.
 
 ## Writing a WebSocket client
 
 A Worker can also establish a WebSocket connection to a remote server. 
 
-We do not currently support the client implementation described above within a Worker. Instead, WebSockets are established by making a fetch request to a URL with the upgrade header set.
+Cloudflare does not currently support the client implementation described above within a Worker. Instead, WebSockets are established by making a fetch request to a URL with the upgrade header set.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/using-websockets.md]:
```suggestion
Cloudflare does not currently support the client implementation described above within a Worker. Instead, WebSockets are established by making a fetch request to a URL with the `Upgrade` header set.
```

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**